### PR TITLE
feat: allow resuming paused instances 

### PR DESCRIPTION
**Pull request by teleivo**
by scaling back to 1. This ignores the number of replicas an instance (deployment, stateful set) had when paused. We currently do not support scaling instances so this assumption is alright for now.

We are not checking whether an instance has been paused as `resume` is idempotent. If we support scaling anything other than 1 replicas this decision can be revisited.

### CODE REVIEW

**Comment by tonsV2** [comment]
Are the updates to the skaffold file and the destroy consumer need for this PR?

File: pkg/instance/handler.go [hunk 1]
```diff
@@ -371,6 +372,64 @@ func (h Handler) Pause(c *gin.Context) {
 	c.Status(http.StatusAccepted)
 }
 
+// Resume paused instance
+func (h Handler) Resume(c *gin.Context) {
+	// swagger:route PUT /instances/{id}/resume resumeInstance
+	//
+	// Resume paused instance
+	//
+	// Resume paused instance...
```
[the remark this answers is not in the document]
Comment (teleivo) [reply pkg/instance/handler.go]: done 😄

**Comment by teleivo** [comment]
> Are the updates to the destroy consumer required for this PR? thinking

No they are not required. Its just that we always have to update this test and add a NOP method to satisfy the mock. I wanted to fix my pain 😋 Are you against that change?

File: pkg/instance/docs.go [hunk 2]
```diff
@@ -34,6 +34,13 @@ type _ struct {
 	ID uint `json:"id"`
 }
 
+// swagger:parameters resumeInstance
```
[the remark this answers is not in the document]
Comment (teleivo) [reply pkg/instance/docs.go]: makes sense, thank you for catching it 😄

**Review by tonsV2 — APPROVED** [review]

**Pull request merged**